legal.rss

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Do we want to be committing this file or is this the equivalent to committing the terms of service word doc? If we don't want to ever commit this file, we should consider adding it to the `.gitignore`

Reply:
This is the template for what's served in the RSS feed so it needs to be committed. It's separate from the terms-of-service word-doc.